### PR TITLE
Auth Infa: init

### PR DESCRIPTION
**Pull request by amarjandu**
just initialize some stuff to get started with.

### CODE REVIEW

File: environment [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+OIDC_GROUP_CLAIM="${OIDC_AUDIENCE}/group"
+
+
+##
+AUTH_TENANT_SECRET_NAME="auth_tenant"
+
+# These are placeholders, please please these in an environment.local to avoid leaks.
+AUTH_TENANT_URL="something.auth0.com"
+AUTH_CLIENT_ID="2333"
+AUTH_CLIENT_SECRet="shhh"
```
Comment (chmreid) [inline environment]:
```suggestion
AUTH_CLIENT_SECRET="shhh"
```

File: environment [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+AUTH_URL=https://auth.ucsc.ucsc-cgp-redwood.org
+OPENID_PROVIDER=https://dev-4lyab62k.auth0.com/
+OIDC_EMAIL_CLAIM="${OIDC_AUDIENCE}/email"
+OIDC_GROUP_CLAIM="${OIDC_AUDIENCE}/group"
+
+
+##
+AUTH_TENANT_SECRET_NAME="auth_tenant"
+
+# These are placeholders, please please these in an environment.local to avoid leaks.
```
Comment (chmreid) [inline environment]: Can split this into an `environment.local.example` in this PR or a future PR

Comment (amarjandu) [reply environment]: i oped to drop that entire file , and use SSM

**Review by chmreid — APPROVED** [review]

File: environment (hunk 1 above)
Comment (amarjandu) [reply environment]: i just removed it, nice try tho :)

File: environment (hunk 2 above)
Comment (chmreid) [reply environment]: nice, the environment-free revolution inches forward

**Pull request merged**